miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultInstrumentService.java

```diff
@@ -123,9 +195,12 @@ private void applyChanges(Instrument target, Instrument source) {
   }
```

Comment:
Please call `target.setServiceRecords()` in the `applyChanges` or `update` function